packages/backend/src/open_payments/grant/model.ts

```diff
@@ -0,0 +1,34 @@
+import { Model, QueryContext } from 'objection'
+
+import { AccessType, AccessAction } from '../auth/grant'
+import { AuthServer } from '../authServer/model'
+import { BaseModel } from '../../shared/baseModel'
+
+export class Grant extends BaseModel {
```

Comment:
Anything we can do here (naming wise) to make it more explicit what the difference between Grant and GrantReference is?

Reply:
What about `GrantReference` -> `ResourceGrant` (or (PaymentPointer)SubresourceGrant)?

Reply:
I'm also tempted to backtrack on this: https://github.com/interledger/rafiki/pull/535#discussion_r954891956

We could instead store `clientId` (soon to be `client` payment pointer #737) on subresources.
Rename `GrantReference` as `OutgoingPaymentGrants` (since it's only really needed to lock on limit checking :point_down:) and drop the `clientId` column
https://github.com/interledger/rafiki/blob/a5c36d7212c6dadc600f574186d5d5f08462c718/packages/backend/src/open_payments/payment/outgoing/service.ts#L255-L256

Reply:
:grimacing: 
- https://github.com/interledger/rafiki/pull/800

Reply:
Renaming is the right move since we only store references to outgoing payment grants. I'll have to look into the other PR to understand why we can drop `clientId`.

Reply:
At the moment, we store references for grants for incoming payments, quotes, and outgoing payments because it is how we associate `cilentId` with those resources.